packages/analytics/modules/analytics/client.ts

```diff
@@ -13,6 +13,37 @@ import { browser } from '@wxt-dev/browser';
 
 const ANALYTICS_PORT = '@wxt-dev/analytics';
 
+// Based on https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/background#browser_support
+function isBackgroundScript(): boolean {
+  if (!browser?.runtime?.getManifest || !location.protocol.endsWith('extension:')) {
+    return false;
+  }
+
+  const { background, background_page: backgroundPageWXT, manifest_version } = browser.runtime.getManifest();
+  // Misconfiguration or Non-WXT Context Check
+  // Force check page attribute for possible configurations
+  const backgroundPageManifest = (background as any)?.page;
+
+  const backgroundPageOrScriptsCheck =
+    location.pathname === '/_generated_background_page.html' ||
```

Comment:
Where is this string coming from?

Reply:
I found it in source code of browsers

1. https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/chromium/chromium%24+_generated_background_page.html&patternType=keyword&sm=0
2. https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/mozilla-firefox/firefox%24++_generated_background_page.html&patternType=keyword&sm=0
3. https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/WebKit/WebKit%24+_generated_background_page.html&patternType=keyword&sm=0